src/__tests__/SearchData.test.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import TableContainer from '../components/pages/Table/TableContainer';
+import { legacy_createStore as createStore } from 'redux';
+import { Provider } from 'react-redux';
+
+import reducers from '../state/reducers';
+
+//anywhere you test antd you must add this window.matchMedia
```

Comment:
Thank you for these comments -- I know how ridiculously complicated researching this was, considering Ant Design's *ahem-not-ahem* stellar documentation and this makes replication an easy thing for the next developer.